src/poprox_recommender/evaluation/generate/worker.py

```diff
@@ -223,11 +223,11 @@ def recommend_batch(
 
     # wrap in a Task to record resource consumption
     with Task("generate-batch", subprocess=True, reset_hwm=True) as task:
-        for request_id in batch:
-            request = dataset.lookup_request(request_id)
+        for rec_id in batch:
+            request = dataset.lookup_request(rec_id)
```

Comment:
This appears to cross two different concepts in a way that makes me suspect something is still wonky in the terminology being used